docs/src/main/asciidoc/README.adoc

```diff
@@ -1,8 +1,7 @@
 = How to Create Quarkus Documentation
 include::./attributes.adoc[]
 
-This guide describes the asciidoc format and conventions that the {project-name} has
-adopted.
+This guide describes the asciidoc format and conventions that the Quarkus has adopted.
```

Comment:
hmm, this was wrong. It should either be `the Quarkus project` or just `Quarkus`